website/src/content/deployment-examples/azure-blob-storage.mdx

```diff
@@ -0,0 +1,143 @@
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<project>.z13.web.core.windows.net/"
+    }
+  }
+}
+```
+
+# Connect Azure Blob Storage with GitHub Actions
+
+One of the ways to deploy to Azure is to use [Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/). This service allows hosting a static website, but this needs to be enabled and properly configured in the service. 
+
+Follow the steps in the [Azure Storage Account creator](https://portal.azure.com/#create/Microsoft.StorageAccount) to create a new storage account.
+
+## Configuring static website hosting
+
+Once the storage account is created, go to `Static website` section of the storage account and provide the following settings:
+* Static website: `enabled`
+* Index document name: `index.html`
+* Error document path: `index.html`
+
+## Generating deployment credentials
+
+Follow [the Azure instruction](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-static-site-github-actions?tabs=userlevel#generate-deployment-credentials) to generate deployment credentials. 
+This step will result with the following JSON object as output in the command line:
+
+```bash
+{
+  "clientId": "598...6fe",
+  "clientSecret": "OfU...cmr",
+  "subscriptionId": "c43...c21",
+  "tenantId": "591...e24",
+  ...
+}
+```
+
+## Configuring GitHub Action workflow
+
+In your repository [configure the GitHub secret](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-static-site-github-actions?tabs=userlevel#configure-the-github-secret) named `AZURE_CREDENTIALS` and paste the whole JSON object from the previous step as the secret value.
+
+As the next step, [add a GitHub Action workflow](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-static-site-github-actions?tabs=userlevel#add-your-workflow). For the basic GitHub Action workflow you might want to use the following content in the `yaml` file or change it according to your needs:
+
+```yaml
+name: Blob storage website CI
+
+on:
+  push:
+        branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Install dependencies
+      run: yarn install --immutable
+
+    - name: Building
+      run: yarn build
+      
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+   
+    - name: Upload to blob storage
+      uses: azure/CLI@v1
+      with:
+        inlineScript: |
+            az storage blob delete-batch --account-name <STORAGE_ACCOUNT_NAME> --auth-mode key -s '$web' # update with your <STORAGE_ACCOUNT_NAME>
+            az storage blob upload-batch --account-name <STORAGE_ACCOUNT_NAME> --auth-mode key -d '$web' -s public/ # update with your <STORAGE_ACCOUNT_NAME>
+
+    - name: logout
+      run: |
+            az logout
+      if: always()
+```
```

Comment:
This workflow is the [modified workflow from Azure tutorial](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-static-site-github-actions?tabs=userlevel#add-your-workflow), but it covers additional steps (installing deps and building the Custom App), deleting old files from the Blob Storage and does not cover CDN setup that I omitted here